src/api/utils/db_utils.py

```diff
@@ -6,6 +6,8 @@
 import uuid
 
 # Third-Party Libraries
+import pymongo
+
```

Comment:
I dont belive mixing in 2 db clients is the best idea, we are already using motor insead of pymongo

Reply:
I moved the function `clear_and_set_default()` to a new file and kept pymongo for it. I realized that what he is doing in the function we don't already have prebuilt functions and didn't want to take the time to write one. I think this works for now.

However, the update_email_template_cache() function can utilize the update_single() function that already exists. I delete that function and replaced with what's already there.